pkg/controller/importconfig/cluster_info_test.go

```diff
@@ -356,7 +356,16 @@ func TestValidateCAData(t *testing.T) {
 		valid            bool
 	}{
 		{
-			name: "CA data is empty",
+			name:          "bootstrap CA data is empty",
+			currentCAData: certData1,
+		},
+		{
+			name:            "current CA data is empty",
+			bootstrapCAData: certData1,
+		},
+		{
+			name:  "bootstrap and current CA data is empty",
+			valid: true,
```

Comment:
Both `bootstrapCAData` and `currentCAData` can be either `nil` or empty slice `[]byte{}`. Please ensure these cases are also covered.